coralproject/talk
# [CORL-2653] Decrement feature tag counts when rejecting a featured comment

## What does this PR do?

Decrement feature tag counts when rejecting a featured comment. Adds the before/after tag update when we remove the featured tag off of a featured comment when we rejecting a featured comment.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes.

## How do I test this PR?

- Feature a comment
- See featured count go up
- Reject the comment
- See featured count go down
 
## How do we deploy this PR?

No special considerations.
